tests/unit_test/channels/mail_channel_test.py

```
@@ -0,0 +1,693 @@
+import asyncio
+import os
+import shutil
+
+import bson
```

Ruff (0.7.0)

`bson` imported but unused

Remove unused import: `bson`

(F401)